scripts/build

```diff
@@ -21,10 +20,24 @@ var template = fs.readFileSync(path.join(__dirname, 'template.js'),
 //  version :: String
 var version = require('../package.json').version;
 
-//  $pathEq = [String] -> a -> Object -> ()
-var $pathEq = R.curry(function(path, expected, obj) {
-  assert.strictEqual(R.path(path, obj), expected);
-});
+//  logWithPrefix = String -> String -> ()
+var logWithPrefix = R.curryN(2, R.pipe(
+  R.concat,
+  R.concat(_, "\n"),
```

Comment:
My bad. :) I've updated to single-quotes and made some changes to appease jshint.